server/views/pages/404.njk

```diff
@@ -0,0 +1,16 @@
+{% extends "../partials/layout.njk" %}
+
+{% set pageId = 'page-not-found' %}
+{% set pageTitle = "Page not found" %}
+
+{% block content %}
```

Comment:
These 2 nunjucks files (`404` and `error`) are copied verbatim from PLP, which means even though technically it's 2 different UIs, from a user point of view the rendering and messaging etc will be consistent. (that will keep the likes of Jo and Karen happy 😁 )